larray/inout/pickle.py

```diff
@@ -55,5 +55,6 @@ def _dump_metadata(self, metadata):
         self.data['__metadata__'] = metadata
 
     def close(self):
+        # FIXME: this is wrong in so many ways. If we only read the file, it will be re-dumped when closed?
```

Comment:
why not override the save() method and move the code below to it? Instead of adding a FIXME.

Reply:
because I didn't write that code and don't know the implications of modifying it. If it's that simple, I will try that.